platform/src/main/java/org/stellar/anchor/platform/configurator/EnvVarsReader.java

```diff
@@ -0,0 +1,3 @@
+package org.stellar.anchor.platform.configurator;
+
+public class EnvVarsReader {}
```

Comment:
This is not being used anywhere, is it? Can we delete?

Reply:
Removed.